partials/favorites.html

```diff
@@ -0,0 +1,6 @@
+<div class="text-center" id="log">
```

Comment:
Make sure this is just doing/showing "Hello Favorites" (basically what you did in contacts/new and contacts/view and login).

Reply:
done